crates/db_schema/replaceable_schema/utils.sql

```diff
@@ -151,3 +151,118 @@ DECLARE
 END;
 $a$;
 
+-- Edit community aggregates to include voters as active users
```

Comment:
These were just copy-pasted, correct? If nothing was changed here, then we should be fine.

Reply:
Yes its all copy-paste, except getting rid of comment_like.post_id usage and adjusting the name.